2016/proposals/maurya-kumar-raj.md

```diff
@@ -0,0 +1,71 @@
+
+### July 20th - August 2nd
+
+### August 3rd - August 16th
+
+### August 17th - August 21th 19:00 UTC
+
+## Future works
+
+* I will continue to work on the Retriever Project and finish some feature requests and i will be involved in fixing some bugs in Retriever.
+* Adding more reliable name resolution APIs.
+* Improving the accuracy of the best name deciding algorithm.
+* Bug fixing the existing and adding more engines for different formats of data. For example - JSON, XML etc.
+
+## Open Source Development Experience
+
+I have worked on few open source projects like Google's tensorflow that deals with Deep Learning and involves the Machine Learning concepts. I also have contribution in fixing bugs in Mozilla community at bugzilla and also contributed as Firefox Student Ambassador.
+
+I have also worked with a start-up from IIT Kanpur,SIDBI Incubation Center, (India) which involved the development of Dynamic UI graph using plot.ly API for Python and Java Script for testing and plotting the large experimental data sets.
+
+I have been the member of NumFocus community and supporting it as a volunteer.
+
+## Academic Experience
+
+I am currently a 3rd year (B.Tech) Computer Science and Engineering student at University Institute of Engineering and Technology, Kanpur, India. I have done some academic projects on database management systems. I am a FOSS enthusiast and like to contribute, collaborate for the same.
+
+This project deals with implementation and automating the process of accessing the data in EcoData Retriever that requires the knowledge Python and database systems like MySql, SqLite, which I am familiar with and have experience as well as done few projects on that technology.
+
+I also have interned at some research organizations like Indian Institute of Science Education Research (IISER) Kolkata, UNESCO.
+Apart from these I have also interned at start-up companies which includes Kanopy Techno Solutions, Pariksha.co, Pollinate Energy, Absentia Virtual Reality.
+
+## Why this project?
+
+While going through the list of organizations and their ideas pages, I was actually looking for the projects which involves the extensive use of database systems, manipulating the large datasets accordingly and I found this project more suitable as well as favorable to me as it involves the usage of almost 80% of the skill sets and technologies that i am already familiar with. Also, my academic as well as non academic projects involved the processing of large data sets using sophisticated database systems accordingly.
+
+I want to be involved in this project because it could help scientists and researchers to do their research and will save their precious time. I will be glad to contribute to this project and help others through this open source projects. So I think it's very meaningful and useful project for me and also i will be learning a lot as well as this project would useful in extending my skills and abilities as well.
```

Comment:
\[this need to be removed  \]
So I think it's very meaningful and useful project for me and also i will be learning a lot as well as this project would useful in extending my skills and abilities as well.
\[or \]
Last, the project will be a stepping stone for expanding my experience, skills and abilities as well